pom.xml

```diff
@@ -18,7 +18,7 @@
     <inceptionYear>2013</inceptionYear>
 
     <properties>
-        <version.org.fenixedu.academic>22.2.0</version.org.fenixedu.academic>
+        <version.org.fenixedu.academic>22.3.0-SNAPSHOT</version.org.fenixedu.academic>
```

Comment:
This version needs to be updated with the release of [ACDM-1658](https://github.com/FenixEdu/fenixedu-academic/pull/1530)